package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "devDependencies": {
+    "@symfony/webpack-encore": "^0.20.0",
+    "bootstrap": "^4.1.3",
+    "date-fns": "^1.29.0",
+    "jquery": "^3.3.1",
```

Comment:
would it make sense to use `jquery-slim` in this spot (as we used the slim variant also before this PR)?

or do some parts of bootstrap require the "full build"?

Reply:
The `jquery-slim` version is not offered as an npm package. The `jquery` package contains a compiled version of `jquery-slim` though, but funnily, if you require that instead of the regular `jquery`, the bundled js is significantly larger.

Bootstrap 4 requires `jquery` in order to run it's javascript features, in addition to `popper` if you want to use popups, tooltips and the like.

As of now, the PackageFilter class (which I ported directly from the previous code) makes heavy use of `jquery`, and I've also introduced the use of Bootstrap's collapsible plugin.

I've been working a bit on this but I think maybe it's better to be done in another PR once this is merged.